README.md

```diff
@@ -52,7 +52,7 @@ You will need the following:
 
 * JVM 8
 * Android SDK and the latest stable Android Studio or IntelliJ
-* Intellij Kotlin plugin with 1.3.71 support
+* Intellij Kotlin plugin with 1.3.71 support (should be included in the latest Android Studio)
 * Mac with Xcode 11+ for the iOS build
```

Comment:
Not here, but at some point we should test and have advice around Xcode 12 (in beta now)